src/_11ty/shortcodes/ArticleCard.js

```diff
@@ -0,0 +1,60 @@
+const filtersDir = `../filters`;
+const articleType = require(`${filtersDir}/articleType.js`);
+const formatDate = require(`${filtersDir}/formatDate.js`);
+const removeHtml = require(`${filtersDir}/removeHtml.js`);
+const truncate = require(`${filtersDir}/truncate.js`);
+
+module.exports = (
+  {
+    data: { author = '', date, image, tags, title = '', locale = '' },
+    templateContent,
+    url,
+  },
+  label = ''
```

Comment:
Suggestion: Perhaps if we think we might need any additional configuration for `ArticleCard`, we might consider its API: `ArticleCard(item, options)`

Just thinking, it could be more self-describing, and leave us open to other options in future w/o having to rework the existing implementations. E.g.

```js
ArticleCard(item, {
  showLabel: true
})
```

Reply:
@adamduncan Good idea.

I'll update that.